mysql-test/suite/tianmu/t/create_table.test

```diff
@@ -0,0 +1,336 @@
+drop table if exists t1,t2,t3,t4,t5;
+drop database if exists mysqltest;
+drop view if exists v1;
+create table t1 (b char(0) not null)ENGINE=tianmu;
+--error 1048
+insert into t1 values (""),(null);
+create table if not exists t1 (b char(0) not null)ENGINE=tianmu;
+--error 1167
+create table t1 (b char(0) not null, index(b))engine=tianmu;
+drop table t1;
+create table `a/a` (a int)engine=tianmu;
+show create table `a/a`;
+create table t1 like `a/a`;
+drop table `a/a`,`t1`;
+--error 1059
+create table `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa` (aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa int)engine=tianmu;
```

Comment:
Try to avoid more than 80 characters per line.

Reply:
This is an exception use case and is therefore designed to verify the table name length